Editor/Window/ConnectToFleetInput.cs

```diff
@@ -73,9 +73,11 @@ private async Task OnAnywhereConnectClicked(string fleetName)
                 var createFleetResponse = await _fleetManager.CreateFleet(fleetName, customLocationResponse.Location)!;
                 if (createFleetResponse.Success)
                 {
-                    _stateManager.AnywhereFleetName = createFleetResponse.FleetName;
-                    _stateManager.AnywhereFleetId = createFleetResponse.FleetId;
-                    _stateManager.AnywhereFleetLocation = customLocationResponse.Location;
+                    var fleetLocationResponse = await _fleetManager.FindFirstFleetLocation(response.FleetId);
+                    _stateManager.AnywhereFleetName = response.FleetName;
+                    _stateManager.AnywhereFleetId = response.FleetId;
+                    _stateManager.AnywhereFleetLocation = fleetLocationResponse.Location;
```

Comment:
```suggestion
                    _stateManager.AnywhereFleetName = createFleetResponse.FleetName;
                    _stateManager.AnywhereFleetId = createFleetResponse.FleetId;
                    _stateManager.AnywhereFleetLocation = customLocationResponse.Location;
```
Revert this change. We already have the location from the create custom location call. This also is using the wrong response name so wouldn't compile